docs/install/index.md

```diff
@@ -11,96 +11,53 @@ myst:
 
 # Install
```

Comment:
The content of this page looks good but I think the headings are a bit confusing, because only half of the material here is about installing Plone (the demos do not require installing anything).

I think it would help if you swapped the "Getting started" and "Install" headings (so Getting Started is at the top as h1 and Install is further down as h2).

This will change the label in the nav from Install to Getting Started. I think that's fine?

Reply:
Swapping headings means renaming the file and its title would change. I don't want that, as most visitors to _documentation_ (not plone.org) do in fact want to install Plone, and only a few realize, "Oh, I don't need to install it, there's a demo. Maybe I should try it before I buy it?"

This is a common issue in many open source projects. Do we tell people "Getting started", or "Install", or demo, or some combination of the above? I think "Getting started" is not a good primary path in documentation because it buries "Install", and I think that "Install" is more important than the generic catch-all "Getting started". Most of the searches in our Matomo stats I see are for "Install" (7) not "Getting started" (1). Finally "Getting started" is not the same as trying a demo. Thinking out loud, I don't think we want a separate demo page, either. I'd prefer to direct people to a single page from which decisions are made, as it is now. plone.org handles this better than documentation could, forcing a visitor to make a decision in its menu.

![Screenshot 2024-01-15 at 6 21 55 AM](https://github.com/plone/documentation/assets/102112/f05d290d-1d76-45f2-a258-6c0191ef5a52)

Reply:
@stevepiercy I find these arguments convincing. We're optimizing for what helps people find the right information, not for the structure that makes sense in my head. :)